code/go/internal/validator/semantic/validate_visualizations_used_by_value.go

```diff
@@ -83,7 +83,7 @@ func anyReference(val interface{}) ([]reference, error) {
 	var references []reference
 	for _, reference := range allReferences {
 		switch reference.Type {
-		case "lens", "visualization", "map":
```

Comment:
You can run the following command to test from a package directory:
```
elastic-package check
```
You can find packages to test in https://github.com/elastic/integrations/

These errors will be displayed only as warnings, unless you set the environment variable `PACKAGE_SPEC_WARNINGS_AS_ERRORS=true`.

Reply:
thx!

Reply:
> Just as a note, this is going to be applied independently of the spec version used in the packages.

I think this is fine, as it will only produce validation errors in v3.